api/ruby/instance-auditing/README.md

```diff
@@ -0,0 +1,13 @@
+# Instance auditor
+
+This script creates an spreadsheet file that will allow you to audit the access of each team and user with all of the Organizations across your GitHub Enterprise instance.
+
+## Getting started
+
+Before running the script, the user who is going to run the script must be on the "Owners" team of every Organization you wish the audit. You can promote all users with Staff Tools access to Owners of each Organization by running [`ghe-org-admin-promote`](https://help.github.com/enterprise/2.1/admin/articles/command-line-utilities/#ghe-org-admin-promote).
```

Comment:
pro tip: use the URL without the version number to always get the latest version: https://help.github.com/enterprise/admin/articles/command-line-utilities/#ghe-org-admin-promote